cg/meta/upload/rnafusion/rnafusion.py

```diff
@@ -21,11 +21,11 @@ def __init__(self, config: CGConfig):
         self.analysis_api: RnafusionAnalysisAPI = RnafusionAnalysisAPI(config)
         super().__init__(config=config, analysis_api=self.analysis_api)
 
-    def upload(self, ctx: click.Context, case: Family, restart: bool) -> None:
+    def upload(self, ctx: click.Context, case_obj: Family, restart: bool) -> None:
```

Comment:
@Vince-janv I did change when you said it first, but then it needs to be refactored for the other pipelines too otherwise their upload fail. I didn't feel like doing refactors that involve other pipelines in this PR, is that ok? Or you feel strong about this?

Reply:
I just refactored the other apis too